akka-stream/src/main/boilerplate/akka/stream/scaladsl/ZipWithApply.scala.template

```diff
@@ -56,17 +65,13 @@ class ZipWith1[[#A1#], O] (zipper: ([#A1#]) ⇒ O) extends GraphStage[FanInShape
 
     setHandler(out, new OutHandler {
       override def onPull(): Unit = {
-        pending = shape.inlets.size
-        if (willShutDown) completeStage()
-        else {
-            [#pull(in0)#
-            ]
-        }
+        pending += shape.inlets.size
```

Comment:
`inlets` is an immutable.Seq so `size` could most definitely be `O(n)` so perhaps not do that for every one of these calls?